### ISSUE TITLE
In microsoft Outlook 2019 /365, NVDA fails to read attachments with the latest alpha

### ISSUE THREAD

**Issue by zstanecic** [nvaccess/nvda#12235]
### Steps to reproduce:
1. Open outlook, and locate the message with the multiple attachments, then head over to the attachments list
2. when navigating with the left and right arrows, you will notice that NVDA is not reading the attachments list
3. the folowing log is in the output
ERROR - eventHandler.executeEvent (10:40:30.988) - MainThread (5588):
error executing event: gainFocus on <NVDAObjects.UIA.UIA object at 0x00CD4F10> with extra args of {}
Traceback (most recent call last):
  File "eventHandler.pyc", line 246, in executeEvent
  File "eventHandler.pyc", line 96, in __init__
  File "eventHandler.pyc", line 105, in next
  File "NVDAObjects\UIA\__init__.pyc", line 1142, in event_gainFocus
  File "NVDAObjects\__init__.pyc", line 1137, in event_gainFocus
  File "NVDAObjects\__init__.pyc", line 1005, in reportFocus
  File "speech\__init__.pyc", line 449, in speakObject
  File "speech\__init__.pyc", line 491, in getObjectSpeech
  File "speech\__init__.pyc", line 405, in getObjectPropertiesSpeech
  File "baseObject.pyc", line 42, in __get__
  File "baseObject.pyc", line 146, in _getPropertyViaCache
  File "NVDAObjects\UIA\__init__.pyc", line 1216, in _get_selectionContainer
  File "comtypesMonkeyPatches.pyc", line 27, in __call__
ValueError: NULL COM pointer access
ERROR - eventHandler.executeEvent (10:40:32.107) - MainThread (5588):
error executing event: gainFocus on <NVDAObjects.UIA.UIA object at 0x055B0130> with extra args of {}
Traceback (most recent call last):
  File "eventHandler.pyc", line 246, in executeEvent
  File "eventHandler.pyc", line 96, in __init__
  File "eventHandler.pyc", line 105, in next
  File "NVDAObjects\UIA\__init__.pyc", line 1142, in event_gainFocus
  File "NVDAObjects\__init__.pyc", line 1137, in event_gainFocus
  File "NVDAObjects\__init__.pyc", line 1005, in reportFocus
  File "speech\__init__.pyc", line 449, in speakObject
  File "speech\__init__.pyc", line 491, in getObjectSpeech
  File "speech\__init__.pyc", line 405, in getObjectPropertiesSpeech
  File "baseObject.pyc", line 42, in __get__
  File "baseObject.pyc", line 146, in _getPropertyViaCache
  File "NVDAObjects\UIA\__init__.pyc", line 1216, in _get_selectionContainer
  File "comtypesMonkeyPatches.pyc", line 27, in __call__
ValueError: NULL COM pointer access
ERROR - eventHandler.executeEvent (10:40:33.206) - MainThread (5588):
error executing event: gainFocus on <NVDAObjects.UIA.UIA object at 0x009D0BB0> with extra args of {}
Traceback (most recent call last):
  File "eventHandler.pyc", line 246, in executeEvent
  File "eventHandler.pyc", line 96, in __init__
  File "eventHandler.pyc", line 105, in next
  File "NVDAObjects\UIA\__init__.pyc", line 1142, in event_gainFocus
  File "NVDAObjects\__init__.pyc", line 1137, in event_gainFocus
  File "NVDAObjects\__init__.pyc", line 1005, in reportFocus
  File "speech\__init__.pyc", line 449, in speakObject
  File "speech\__init__.pyc", line 491, in getObjectSpeech
  File "speech\__init__.pyc", line 405, in getObjectPropertiesSpeech
  File "baseObject.pyc", line 42, in __get__
  File "baseObject.pyc", line 146, in _getPropertyViaCache
  File "NVDAObjects\UIA\__init__.pyc", line 1216, in _get_selectionContainer
  File "comtypesMonkeyPatches.pyc", line 27, in __call__
ValueError: NULL COM pointer access

### Actual behavior:
NVDA doesn't read attachments names and Error sound is present
### Expected behavior:
NVDA should read attachments list
#### NVDA installed/portable/running from source:
installed
#### NVDA version:
NVDA version alpha-22148,9a1cca54

#### Windows version:
Windows 10 20H2 (10.0.19042) workstation

#### Name and version of other software in use when reproducing the issue:
MSO (16.0.13801.20288) (64-bit) 
#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Not applicable, as the regression is recent
#### If add-ons are disabled, is your problem still occurring?
yes
#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
not applicable

**Comment by lukaszgo1** [comment]
@zstanecic Is this still reproducible for you with latest Alpha?

**Comment by zstanecic** [comment]
Hi,

I am wondering why this was not automatically closed with one of the recent commits.

This was recently fixed, so it is worth clising…

**Comment by lukaszgo1** [comment]
@zstanecic Please close it then.
